packages/appstore/src/components/loader/cfd-app-skeleton/cfdAppskeleton.scss

```diff
@@ -0,0 +1,31 @@
+@import '../skeletonLoader.scss';
+
+.cfd-app-skeleton-loader {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+
+    &__header {
+        display: flex;
+        flex-direction: row;
```

Comment:
I believe you can remove: `flex-direction: row` because when you define display as `flex` it will default the flex direction to `row`.